conda.recipe/post-link.bat

```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_conda_kernels.install --enable --prefix="%PREFIX%"
+"%PREFIX%\python" -m nb_conda_kernels.install --enable --prefix="%PREFIX%"
```

Comment:
@malev ^^^

@bollwyvl looking into nbpresent, it seems you use it a reliable way... https://github.com/Anaconda-Server/nbpresent/blob/master/conda.recipe/bld.bat

Did you validated that on win for nbpresent? I mean... it is working OK?